### PR TITLE
Addon Bytebase

### PR DESCRIPTION
**Pull request by MdSahil-oss**
### Description of your changes
Adds bytebase as an addon
Fixes https://github.com/kubevela/kubevela/issues/5878

### Checklist

I have:

- [x] Title of the PR starts with type (e.g. `[Addon]` , `[example]` or `[Doc]`).
- [x] Updated/Added any relevant [documentation](https://kubevela.io/docs/reference/addons/overview) and [examples](https://github.com/kubevela/catalog/tree/master/examples).
- [x] New addon should be put in [experimental](https://github.com/kubevela/catalog/tree/master/experimental/addons).
- [x] Update addon should modify the `version` in `metadata.yaml` to generate a new version.

####  Verified Addon promotion rules

If this pr wants to promote an experimental addon to verified, you must check whether meet these conditions too:
  - [x] This addon must be tested by addon's [e2e-test](./test/e2e-test/addon-test) to guarantee this addon can be enabled successfully.
  - This addon must have some basic but necessary information.
    - [x] An accessible icon url and source url defined in addon's `metadata.yaml`.
    - [x] A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
    - [ ] Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
    - [ ] It's more likely to be accepted if useful examples provided in example [dir](examples/).

### CODE REVIEW

File: experimental/addons/bytebase/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# bytebase
+
+This is an addon template. Check how to build your own addon: https://kubevela.net/docs/platform-engineers/addon/intro
```
Comment (wonderflow) [inline experimental/addons/bytebase/README.md:3]: tips: we'd better guide users to bytebase UI console at the end of the readme

**Comment by MdSahil-oss** [comment]
@wonderflow Please review this addon again.

File: experimental/addons/bytebase/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Use bytebase
+
+Verify Pods of the bytebase running or not.
+
+```shell
+$ kubectl get po  -n prod  -o wide
+NAME         READY   STATUS    RESTARTS   AGE   IP             NODE       NOMINATED NODE   READINESS GATES
+bytebase-0   1/1     Running   0          27m   10.244.3.148   minikube   <none>           <none>
+```
```
Comment (wonderflow) [inline experimental/addons/bytebase/README.md:37]: use vela status to confirm?

File: experimental/addons/bytebase/template.cue [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+					version:         parameter.chartVersion
+					targetNamespace: parameter.namespace
+					values: {
+						bytebase: {
+							option: {
+								port:           parameter.bytebasePort,
+								"external-url": parameter.externalURL,
+								pg:             parameter.postgresURL
+							},
+							version: parameter.version
```
Comment (wonderflow) [inline experimental/addons/bytebase/template.cue:34]: can we use the deployment yaml instead of helm chart?

Comment (MdSahil-oss) [reply experimental/addons/bytebase/template.cue]: Done 👍🏽

**Review by tianzhou — DISMISSED** [review]
LGTM. @MdSahil-oss thanks for adding Bytebase to kubevela !

File: experimental/addons/bytebase/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+```shell
+$ kubectl get svc -n bytebase
+NAME                           TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+bytebase-entrypoint            LoadBalancer   10.96.143.226   <pending>     443:31537/TCP    40m
+bytebase-nodeport-entrypoint   NodePort       10.104.52.56    <none>        8080:32363/TCP   80m
+
+# Get url to access UI
+$ minikube service -n bytebase --url bytebase-nodeport-entrypoint
+http://192.168.49.2:32363
```
[the remark this answers is not in the document]
Comment (MdSahil-oss) [reply experimental/addons/bytebase/README.md:51]: Done 👍🏽

File: experimental/addons/bytebase/README.md (hunk 2 above)
Comment (MdSahil-oss) [reply experimental/addons/bytebase/README.md:37]: done 👍🏽

**Review by wonderflow — APPROVED** [review]
great job!

File: experimental/addons/bytebase/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+icon: "https://raw.githubusercontent.com/bytebase/bytebase/main/frontend/public/favicon.ico"
+url: "https://www.bytebase.com/"
+
+tags:
+- bytebase
+
+invisible: false
+
+dependencies:
+  - name: fluxcd
```
Comment (wonderflow) [inline experimental/addons/bytebase/metadata.yaml:13]: we can remove this dependencies now

**Review by wonderflow — APPROVED** [review]

File: experimental/addons/bytebase/metadata.yaml (hunk 5 above)
Comment (MdSahil-oss) [reply experimental/addons/bytebase/metadata.yaml]: Done 👍🏽

**Comment by MdSahil-oss** [comment]
@wonderflow I've removed fluxcd dependency.

**Review by wonderflow — APPROVED** [review]

**Pull request merged**